umm-spec-lib/src/cmr/umm_spec/migration/version/collection.clj

```diff
@@ -221,6 +221,21 @@
           (update-in [:SpatialExtent :OrbitParameters] assoc :SwathWidth swath-width :Period period)))
     collection))
 
+(defn- migrate-tiling-id-systems-down
+  "Migrate one TilingIdentificationSystem down if all coordinates' min and max values convert form string to number, 
+   otherwise remove entire tiling identification system (return nil)"
+  [element]
+  (if (and (util/str->num (get-in element [:Coordinate1 :MinimumValue]))
```

Comment:
This could be a `when` and drop the else `nil`

Reply:
yes that's better, done.